packages/commonjs/src/index.js

```diff
@@ -0,0 +1,180 @@
+      let resolveId = id;
+      let resolvedId;
+
+      if (isCore(id)) {
+        // resolve will not find npm modules with the same name as
+        // core modules without a trailing slash. Since core modules
+        // must be external, we can assume any core modules defined
+        // here are npm modules by that name.
+        resolveId += '/';
+      }
+
+      try {
+        resolvedId = nodeResolveSync(resolveId, { basedir: process.cwd() });
+      } catch (err) {
+        resolvedId = resolve(id);
+      }
+
+      // Note: customNamedExport's keys must be normalized file paths.
+      // resolve and nodeResolveSync both return normalized file paths
+      // so no additional normalization is necessary.
+      customNamedExports[resolvedId] = options.namedExports[id];
+
+      if (existsSync(resolvedId)) {
+        const realpath = realpathSync(resolvedId);
+        if (realpath !== resolvedId) {
+          customNamedExports[realpath] = options.namedExports[id];
+        }
+      }
+    });
+  }
+
+  const esModulesWithoutDefaultExport = new Set();
+  const esModulesWithDefaultExport = new Set();
+  // TODO maybe this should be configurable?
+  const allowDynamicRequire = !!options.ignore;
+
+  const ignoreRequire =
+    typeof options.ignore === 'function'
+      ? options.ignore
+      : Array.isArray(options.ignore)
+      ? (id) => options.ignore.includes(id)
+      : () => false;
+
+  const resolveId = getResolveId(extensions);
+
+  const sourceMap = options.sourceMap !== false;
+
+  function transformAndCheckExports(code, id) {
+    const { isEsModule, hasDefaultExport, ast } = checkEsModule(this.parse, code, id);
+    if (isEsModule) {
+      (hasDefaultExport ? esModulesWithDefaultExport : esModulesWithoutDefaultExport).add(id);
+      return null;
+    }
+
+    // it is not an ES module but it does not have CJS-specific elements.
+    if (!hasCjsKeywords(code, ignoreGlobal)) {
+      esModulesWithoutDefaultExport.add(id);
+      return null;
+    }
+
+    const normalizedId = normalize(id);
+
+    const transformed = transformCommonjs(
+      this.parse,
+      code,
+      id,
+      this.getModuleInfo(id).isEntry,
+      ignoreGlobal,
+      ignoreRequire,
+      customNamedExports[normalizedId],
+      sourceMap,
+      allowDynamicRequire,
+      ast
+    );
+    if (!transformed) {
+      esModulesWithoutDefaultExport.add(id);
+      return null;
+    }
+
+    return transformed;
+  }
+
+  return {
+    name: 'commonjs',
+
+    buildStart() {
+      const [major, minor] = this.meta.rollupVersion.split('.').map(Number);
+      const minVersion = peerDependencies.rollup.slice(2);
+      const [minMajor, minMinor] = minVersion.split('.').map(Number);
+      if (major < minMajor || (major === minMajor && minor < minMinor)) {
+        this.error(
+          `Insufficient Rollup version: "rollup-plugin-commonjs" requires at least rollup@${minVersion} but found rollup@${this.meta.rollupVersion}.`
```

Comment:
there are a few lingering `rollup-plugin-commonjs` instances in the directory that should be updated. note that we want to leave in place the references to isssues on the old repo.